test/e2e/conformance_test.go

```diff
@@ -147,9 +147,6 @@ var _ = Describe("Conformance Tests", func() {
 			// Conformance for windows doesn't require any linux worker machines.
 			// The templates use WORKER_MACHINE_COUNT for linux machines for backwards compatibility so clear it
 			linuxWorkerMachineCount = 0
-
-			// Windows e2e tests require HPAContainerMetrics featuregate
-			Expect(os.Setenv("K8S_FEATURE_GATES", "HPAContainerMetrics=true")).To(Succeed())
```

Comment:
@jsturtevant if you're willing to take on the add'l test config burden of this, I do think it makes more sense there than hard-coding it in CAPZ E2E

Lemme know if I can help review a test-infra PR!